src/codegen/abi.js

```diff
@@ -520,7 +535,19 @@ module.exports = class AbiCodeGenerator {
           );
           return ${
             simpleReturnType
-              ? typesCodegen.ethereumValueToAsc('result[0]', outputs.get(0).get('type'))
+              ? typesCodegen.ethereumValueToAsc(
+                'result[0]', 
+                outputs.get(0).get('type'), 
+                outputs.get(0).get('type')==='tuple[]' 
```

Comment:
The formatting is off here (no spaces before/after the `===`). Make sure you run prettier against the changes.